src/libs/azure-utils.js

```diff
@@ -7,11 +7,10 @@ export const defaultAzureRegion = 'eastus'
 
 export const defaultAzureComputeConfig = {
   machineType: defaultAzureMachineType,
-  diskSize: defaultAzureDiskSize,
-  region: defaultAzureRegion
+  diskSize: defaultAzureDiskSize
 }
 
-//TODO: this should be fleshed out once azure region isn't limited to workspace region
+//TODO: this should be fleshed out once azure region isn't limited to workspace region. Remove this once all the references are gone.
 export const azureRegions = { eastus: { flag: '🇺🇸', label: 'East US' } }
```

Comment:
didn't remove this directly becuz it's being used by other files

Reply:
We do plan to use this to show the actual region on the Workspace Dashboard (for getting the label and flag). We will need to add more regions to the dict though.